docs/user_manual.md

```diff
@@ -133,6 +133,8 @@ example that uses all the available configuration keys with an explanation of ea
 
 ```json
 {
+ "addedCapabilities" : [ "IPC_LOCK", "SYSLOG" ],
+ "droppedCapabilities" : [ "SYS_BOOT", "KILL" ],
```

Comment:
forgive the nitpick, but how about `"addCapabilities" / "dropCapabilities"` in the job json so it's not using past tense?